### ISSUE TITLE
NTDOY doesn't show values after market closes

### ISSUE THREAD

**Issue by ngocphamm** [achannarasappa/ticker#97]
**Describe the bug**
I'm not sure if this is because of the particular stock (NTDOY) or what. Just weird that after market closes the values are all 0s, so my lots show incorrect numbers, too.
If I restart ticker, it's showing again, until the next time it refreshes (by the interval set), it's all 0s again.

**To Reproduce**
1. Start `ticker` with lots containing `NTDOY`
```
lots:
  - symbol: "NTDOY"
    quantity: xxx
    unit_cost: xx.xx
```
2. See all zeros for the symbol, once the market closes
3. If `ticker` is left open, once the market opens again the next day, values will come back again

**Expected behavior**
Values are not set to 0s after market closes.

**Screenshots**
![CleanShot 2021-02-11 at 08 09 42@2x](https://user-images.githubusercontent.com/395123/107641540-c2b13700-6c41-11eb-82d5-8e9a422b79e8.png)

**Environment (please complete the following information):**
 - OS: macOS
 - Terminal: Alacritty, tmux 3.1c
 - Terminal Version: 0.7.1
 - Font: Jetbrains Mono
 - ticker Version: 3.0.7

**Comment by achannarasappa** [comment]
Assuming you took this just now, I can see this stock is in the `PRE` market state but there is no pre-market price. Definitely a bug - thanks for reporting

**Comment by ngocphamm** [comment]
Yup the screenshot was taken at the time I created the issue.

**Comment by danemacmillan** [comment]
I've noticed something similar, so it's probably relevant to add. Leaving `ticker` to run overnight--it probably doesn't need all night--will result in all my lots, values, percentages, etc to be either some random negative number, or no number at all. In my case, there are three stocks, all on the TSX Venture exchange (TSX.V).

I've added a screenshot of just the summary line, but the price differences and percentage for the day for each row also become all negative.

<img width="474" alt="Screen Shot 2021-02-11 at 9 29 05 AM" src="https://user-images.githubusercontent.com/4390491/107650641-b4b4e380-6c4c-11eb-8064-48a9244f02f8.png">

As soon as the markets open, all the values are corrected and displayed properly.
